openedx/openedx-platform
# Moved the ID used in the google-site-verification meta tag to settings

This PR Moves the ID string used in the google-site-verification meta tag in lms/templates/main.html to settings so that it can be overridden from the site_configuration as well as lms.env.json. Setting it to an empty string omits the tag completely.

**JIRA tickets**: [OSPR-2319](https://openedx.atlassian.net/browse/OSPR-2319)

**Merge deadline**: "None"

**Testing instructions**:

1. Visit the LMS homepage.
2. View page source and search for "google-site-verification". it should find a meta tag:
`<meta name="google-site-verification" content="_mipQ4AtZQDNmbtOkwehQDOgCxUUV2fb_C0b6wbiRHY" />`
3. To override the content, provide a value for GOOGLE_SITE_VERIFICATION_ID in site_configuration and restart the server (or wait for cache to timeout).

**Reviewers**
- [ ] edX reviewer[s] @nedbat 

**site_configuration override**
{
  "GOOGLE_SITE_VERIFICATION_ID":""
}
